sdk/cosmos/azure-cosmos-spark-account-data-resolver-sample/sample/AccountTokenResolverSample.ipynb

```diff
@@ -85,54 +81,55 @@
     "    \n",
     "chars = string.ascii_letters\n",
     "data = [\\\n",
-    "    (str(uuid.uuid4()), pk, \"\", none, 0, random_string_generator(16000, chars)),\\\n",
-    "    (str(uuid.uuid4()), pk, \"\", none, 0, random_string_generator(random.randint(16000, 170000), chars)),\\\n",
+    "    (str(uuid.uuid4()), pk, \"\", None, 0, random_string_generator(16000, chars)),\\\n",
+    "    (str(uuid.uuid4()), pk, \"\", None, 0, random_string_generator(random.randint(16000, 170000), chars)),\\\n",
     "  ]\n",
     "\n",
-    "schema = structtype([ \\\n",
-    "    structfield(\"id\",stringtype(),false), \\\n",
-    "    structfield(\"pk\",stringtype(),false), \\\n",
-    "    structfield(\"emptycolumn\",stringtype(),true), \\\n",
-    "    structfield(\"nullcolumn\",stringtype(),true), \\\n",
-    "    structfield(\"defaultcolumn\",integertype(),true), \\\n",
-    "    structfield(\"largecolumn\", stringtype(), true)\\\n",
+    "schema = StructType([ \\\n",
+    "    StructField(\"id\",StringType(),False), \\\n",
+    "    StructField(\"pk\",StringType(),False), \\\n",
+    "    StructField(\"emptycolumn\",StringType(),True), \\\n",
+    "    StructField(\"nullcolumn\",StringType(),True), \\\n",
+    "    StructField(\"defaultcolumn\",IntegerType(),True), \\\n",
+    "    StructField(\"largecolumn\", StringType(), True)\\\n",
     "  ])\n",
     " \n",
-    "df = spark.createdataframe(data=data,schema=schema)\n",
-    "df.printschema()\n",
+    "df = spark.createDataFrame(data=data,schema=schema)\n",
+    "df.printSchema()\n",
     "df.show(truncate=100)\n",
     "\n",
     "writecfg = {\n",
     "  \"spark.cosmos.accountendpoint\": cosmosendpoint,\n",
-    "  \"spark.cosmos.database\": \"test\",\n",
-    "  \"spark.cosmos.container\": \"testitems\",\n",
+    "  \"spark.cosmos.accountDataResolverServiceName\": accountDataResolverName,\n",
+    "  \"spark.cosmos.database\": \"Test\",\n",
+    "  \"spark.cosmos.container\": \"TestItems\",\n",
     "  \"spark.cosmos.write.strategy\": \"itemappend\",\n",
     "  \"spark.cosmos.write.bulk.enabled\": \"true\",  \n",
     "  \"cosmos.auth.sample.enabled\": \"true\",\n",
     "  # masterkey\n",
-    "  #\"cosmos.auth.sample.authtype\": \"masterkey\",\n",
+    "  #\"cosmos.auth.sample.authType\": \"masterkey\",\n",
     "  #\"cosmos.auth.sample.key.secret\": cosmosmasterkey,\n",
     "  #\n",
     "  # aad auth with managed identity\n",
-    "  #\"cosmos.auth.sample.authtype\": \"managedidentity\",\n",
-    "  #\"cosmos.auth.sample.tenantid\": \"72f988bf-86f1-41af-91ab-2d7cd011db47\",\n",
-    "  #\"cosmos.auth.sample.subscriptionid\": \"8fba6d4f-7c37-4d13-9063-fd58ad2b86e2\",\n",
+    "  #\"cosmos.auth.sample.authType\": \"managedidentity\",\n",
+    "  #\"cosmos.auth.sample.tenantId\": \"72f988bf-86f1-41af-91ab-2d7cd011db47\",\n",
```

Comment:
I am curious, is there no other way than hard coding these tenantId and subscriptionId?

Reply:
Sure - could be wired up as parameter - but since this is just a sample and the Guid is harmless, I don't see a problem with it.